generate/seccomp/consts.go

```diff
@@ -0,0 +1,7 @@
+package seccomp
+
+const (
+	overwrite = "overwrite"
+	appnd     = "append"
```

Comment:
On Thu, Jul 21, 2016 at 11:41:09AM -0700, Grant Seltzer Richman wrote:
> > @@ -0,0 +1,7 @@
> > +package seccomp
> > +
> > +const (
> > +	overwrite = "overwrite"
> > +	appnd     = "append"
> 
> append is a reserved word

APPEND?  (I think Go is case sensitive, and CAPS for package constants
is idiomatic in Python, although I'm not sure about Go).